ansible_bender/__main__.py

```diff
@@ -0,0 +1,5 @@
+from cli import main
```

Comment:
Plz use absolute imports.
```suggestion
from .cli import main
```